PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -301,6 +306,20 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
     }
 }
 
++ (nonnull PBMJsonDictionary *)getRenderersJson {
+    PBMMutableJsonDictionary * const sdk = [PBMMutableJsonDictionary new];
+    NSMutableArray *renderersArray = [NSMutableArray array];
+    NSArray *renderers = [[PrebidMobilePluginRegister shared] getAllPlugins];
+    for (id<PrebidMobilePluginRenderer> renderer in renderers) {
```

Comment:
SDK should send only custom renderers in the request. 

I see two renderers in the request : 

```
"renderers": [{
    "name": "SampleCustomRenderer",
    "version": "1.0.0"
}, {
    "name": "PrebidRenderer",
    "version": "2.2.3"
}]
```